jekyll/_cci2/circleci-images.md

```diff
@@ -252,6 +252,14 @@ The following packages are installed via `curl` or other means.
 - [dockerize](https://github.com/jwilder/dockerize)
 - [jq](https://stedolan.github.io/jq/)
 
+
+## What Isn't Available
```

Comment:
```suggestion
## Out of Scope
```

Reply:
Maybe? Not sure this is quite right...

Reply:
Umm. I'm still not sure but I think it's better than mine.